src/content/en/updates/posts/2015/10/chrome-47-webrtc.markdown

```diff
@@ -0,0 +1,90 @@
+featured_image: /web/updates/images/2015-10-28-chrome-47-webrtc/featured.png
+---
+
+<p class="intro">Chrome 47 includes several significant WebRTC enhancements and updates.</p>
+
+### Record video from your web apps
+
+The `MediaStreamRecorder` API has long been the top chromium.org request, with over 2500
+stars. Media recording has now been added to Chrome behind the experimental Web Platform features flag &mdash; though it's desktop only (and video-only) for the moment. This allows you to record and play back or download video. There is a simple demo [here](https://rawgit.com/Miguelao/demos/master/mediarecorder.html) and you can find out more from the [discuss-webrtc announcement](https://groups.google.com/forum/?#!msg/discuss-webrtc/n11m846oV4I/0b3ycjmjCAAJ).
+
+### Audio output device selection
+
+`MediaDevices.enumerateDevices()` has been released: more details details in [Issue
+504280](https://crbug.com/504280). You can now enumerate audio output devices in
+addition to the audio input and video input devices that `MediaStreamTrack.getSources()` already provides. You can find out more about how to use it
+in [this update](https://developers.google.com/web/updates/2015/10/media-devices).
+
+Default communications device support on Windows has been added ([Issue
+525443](http://crbug.com/525443)). This means that when enumerating audio
+devices on Windows, there will be an additional entry for the communications
+device whose ID will be 'communications'.
+
+Device IDs for the default audio device (and communications on Windows) will no
+longer be hashed ([Issue 535980](http://crbug.com/535980)).  Instead, two
+reserved IDs, 'default' and 'communications' are supported and are the same
+across all security origins. Device labels will be translated to the browser
+locale so developers should not expect labels to have a predetermined value.
+Video rendering accuracy has been improved by propagating the capture timestamp
+all the way to the rendering algorithm, where the right vsync can be chosen
+based on that. For Windows platform the capture timestamp are also more accurate
+in 47.
+
+### Proxy handling
+
+Chrome 47 adds a new preference to force WebRTC traffic to be sent through a local proxy server, if one is configured, which is important for some users browsing via a VPN. This means that the WebRTC application will only see the proxy IP address. Be aware that this will hurt application performance, and won't work at all unless the application supports TURN/TCP or ICE-TCP. Look for a new version of the [WebRTC Network Limiter Extension](https://chrome.google.com/webstore/detail/webrtc-network-limiter/npeicpdbkakmehahjeeohfdhnlpdklia) soon to provide a UI for this preference.
+
+### ...and more
+
+Data channel throughput has been greatly improved for high latency connections.
+
+We will gradually roll out support for DTLS 1.2 in the M47 timeframe.
+
+VP9 did not make it for this release, nor did H.264 support. Work on these
```

Comment:
Suggested rewrite: "Support for neither VP9 nor H.264 video formats is in this release."